app/scripts/directives/showGroup.js

```diff
@@ -0,0 +1,55 @@
+import template from 'views/components/showGroup.html';
+
+angular.module('confRegistrationWebApp')
+  .component('showGroup', {
+    templateUrl: template,
+    bindings: {
+      modalInstance: '<',
+      resolve: '<'
+    },
+    controller: [function(){
```

Comment:
Does this need to be wrapped in an array? I think we have a babel plugin to write the $inject stuff so manually listing what needs to be injected probably isn't needed.